### PR TITLE
Read from cache for corp_types. Add in cache to pay-queue.

### PR DESCRIPTION
**Pull request by seeker25**
Use this when determining a topic:

image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

File: pay-api/src/pay_api/utils/util.py [hunk 1]
```diff
@@ -257,11 +259,9 @@ def cents_to_decimal(amount: int):
...
 def get_topic_for_corp_type(corp_type: str):
     """Return a topic to direct the queue message to."""
-    no_queue_message_types = [CorpType.BTR.value, CorpType.CSO.value, CorpType.ESRA.value, 
-                              CorpType.MHR.value, CorpType.PPR.value, CorpType.RPT.value, CorpType.VS.value]
-
-    if corp_type in no_queue_message_types:
-        return None
-    if corp_type == CorpType.NRO.value:
+    product_code = CodeService.find_code_value_by_type_and_code(Code.CORP_TYPE.value, corp_type).get('product')
+    if product_code == Product.NRO.value:
```
Comment (seeker25) [inline pay-api/src/pay_api/utils/util.py:263]: only need NRO or BUSINESS for now.

**Review by ochiu — APPROVED** [review]
Looks good!

File: pay-queue/src/pay_queue/__init__.py [hunk 2]
```diff
@@ -58,4 +59,19 @@ def create_app(run_mode=os.getenv('DEPLOYMENT_ENV', 'production')) -> Flask:
 
     register_endpoints(app)
 
+    build_cache(app)
     return app
+
+
+def build_cache(app):
```
Comment (seeker25) [inline pay-queue/src/pay_queue/__init__.py:66]: Will see if I can use this from pay-api

**Comment by seeker25** [comment]
Should use corp type for NRO

**Pull request merged**